truenas/webui
# NAS-116718 / 22.12 / Not all fields are rendered in Launch Docker Image

Testing:
1. Go to Apps in Angelfish and Bluefin.
2. Click Launch Docker Image on both.
3. Compare the two and Check if there is missing field.
